google/cloud/forseti/services/dao.py

```diff
@@ -259,22 +259,40 @@ def define_model(model_name, dbengine, model_seed):
                primary_key=True),
     )
 
+    def get_string_by_dialect(db_dialect, column_size):
+        """Get Sqlalchemy String by dialect.
+
+        Args:
+            db_dialect (String): The db dialect.
+            column_size (Integer): The size of the column.
+
+        Returns:
+            String: Sqlalchemy String.
+        """
+        if db_dialect.lower() == 'sqlite':
+            return String(column_size)
+        else:
+            return String(column_size, collation='utf8mb4_bin')
+
+    # Sqlite doesn't support collation type, need to define different
```

Comment:
This comment should be inside the get_string_by_dialect().  Please also add a comment that this is to support case sensitivity.